app/Party.php

```diff
@@ -442,19 +309,6 @@ public function scopeAttendingOrAttended($query, $userids = null) {
         return $query;
     }
 
-    public function scopeInvitedNotConfirmed($query, $userids = null) {
```

Comment:
Definitely not used?  Seems like a recent addition.

Reply:
Definitely not - searching for InvitedNotConfirmed finds nowt.